app/src/main/kotlin/com/getstrm/pace/api/ProcessingPlatformsApi.kt

```diff
@@ -55,6 +57,14 @@ class ProcessingPlatformsApi(
         processingPlatformsService.getBlueprintPolicy(request)
 
     override suspend fun getLineage(request: GetLineageRequest): GetLineageResponse {
-        return processingPlatformsService.getLineage(request)
+        val lineage = processingPlatformsService.getLineage(request)
+        val withDataPolicyInfo =
+            dataPolicyService.determineLineageDataPolicies(lineage.lineageSummary)
+        val enriched =
```

Comment:
Can you move this to the `ProcessingPlatformsService` class? The API layer should be as dumb as possible.

Reply:
The problem is that doing so will create a cyclical dependency between ProcessingPlatformsService and DataPoliciesService. I know this is a codesmell, but would rather break this up in a separate ticket.

Reply:
Ticket created
https://linear.app/strmprivacy/issue/PACE-119/code-smell